content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,66 @@
+## Installation
+
+- **Added** `global.proxy.resources.limits.memory` to allow users to limit the memory of sidecar proxies.
```

Comment:
We just changed the default I think

Reply:
Then let's remove this from the release notes unless @rshriram objects

Reply:
Might be worth calling out the changed defaults AND that it's configurable through installation

Reply:
oh, i guess the diff tool was not accurate, I will remove it.

Reply:
added back